### PR TITLE
Typo: pipe should be pump

### PR DESCRIPTION
**Pull request by JaimeStill**
### Description

In the description of the `ReadableStream`  processing pattern, step 4 should specify the `pump` function, not `pipe`.

### Motivation

To avoid confusion for future readers.

### Additional details

N/A

### Related issues and pull requests

N/A

### CODE REVIEW

File: files/en-us/web/api/streams_api/using_readable_streams/index.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ This is the standard pattern you'll see when using stream readers:
 1. You write a function that starts off by reading the stream.
 2. If there is no more stream to read, you return out of the function.
 3. If there is more stream to read, you process the current chunk then run the function again.
-4. You keep chaining the `pipe` function until there is no more stream to read, in which case step 2 is followed.
+4. You keep chaining the `pump` function until there is no more stream to read, in which case step 2 is followed.
```
Comment (bsmth) [inline files/en-us/web/api/streams_api/using_readable_streams/index.md:143]:
```suggestion
4. You keep chaining the `pump()` function until there is no more stream to read, in which case step 2 is followed.
```

**Review by bsmth — APPROVED** [review]
Super, thank you!

**Comment by sideshowbarker** [comment]
Jaime, nice catch, congrats on landing your first docs change here — welcome aboard 🎉

**Comment by JaimeStill** [comment]
No worries, glad to help! Appreciate what you all are doing, MDN has been a vital reference source for me.

**Pull request merged**